src/wp-admin/css/site-icon.css

```diff
@@ -2,55 +2,154 @@
   28.0 - Site Icon
 ------------------------------------------------------------------------------*/
 
-.site-icon-preview .favicon-preview {
-	margin: 5px 0 20px;
-	overflow: hidden;
+.site-icon-preview {
+	direction: initial;
+	display: flex;
+	height: 60px;
+	padding: 8px 0 0 8px;
+	align-items: flex-start;
 	position: relative;
-	max-width: 180px;
+	overflow: hidden;
+	box-sizing: border-box;
+	border: 1px solid #8c8f94;
+	border-radius: 4px;
+	background-color: #fff;
+	width: 275px;
+	margin: 0 0 16px 0;
 }
 
-.site-icon-preview .favicon,
-.site-icon-preview .browser-title {
-	height: 16px;
-	left: 88px;
-	overflow: hidden;
-	position: absolute;
-	top: 16px;
+.site-icon-preview.settings {
+	height: 88px;
+	padding: 16px 0 0 16px;
+	width: 350px;
 }
 
-.site-icon-preview .favicon {
-	width: 16px;
+.site-icon-preview.crop {
+	width: 258px;
+	height: 100%;
+	display: grid;
+	grid-template-columns: 8px 1fr;
+	grid-template-rows: 64px 1fr;
+	padding-left: 0;
+	row-gap: 16px;
+	direction: inherit;
 }
 
-.site-icon-preview .browser-title {
-	left: 109px;
-	width: 72px;
-	white-space: nowrap;
+.site-icon-preview.hidden {
+	visibility: none;
```

Comment:
Not sure `visibility` has a `none` value. Anyways, this is probably unnecessary because of `display: none;` two lines below?

Reply:
Removed line 39 and 40.